src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -379,7 +381,9 @@
                         "login": "Login",
                         "repository": "Repository",
                         "totalItems": "In total {{count}} item(s)",
-                        "error": "There was an error while loading the affected Students for this feedback."
+                        "error": "There was an error while loading the affected Students for this feedback.",
+                        "levinsteinInfoTitle": "Group feedback feature enabled",
+                        "levinsteinInfoDescription": "The amount of students affected by the grouped feedback might be higher because only affected students for up to 5 feedbackdetails is sent as more is not really necessary to analyse a common grouped feedback and would lead to performance overhead."
```

Comment:
```suggestion
                        "levenshteinInfoTitle": "Group feedback feature enabled",
                        "levenshteinInfoDescription": "The amount of students affected by the grouped feedback might be higher because only affected students for up to 5 feedbackdetails is sent as more is not really necessary to analyse a common grouped feedback and would lead to performance overhead."
```